### PR TITLE
ballot proofing adapted to ms-either-neither

### PR DESCRIPTION
**Pull request by benadida**

### CODE REVIEW

**Review by eventualbuddha — APPROVED** [review]

File: client/src/screens/BallotScreen.tsx [hunk 1]
```diff
@@ -200,12 +201,41 @@ const BallotScreen = () => {
...
+                  dangerouslySetInnerHTML={{
+                    __html: DOMPurify.sanitize(contest.description),
+                  }}
+                />
+                <ul>
+                  <li>Yes</li>
+                  <li>No</li>
+                </ul>
+              </React.Fragment>
+            ) : null}
```
Comment (beausmith) [inline client/src/screens/BallotScreen.tsx:218]: Any reason you're using a ternary vs expression:
```
{contest.type === 'yesno' && ( ... )}
```

Comment (benadida) [reply client/src/screens/BallotScreen.tsx:218]: no good reason, was just mimicking previous code structure in a rush.

Comment (eventualbuddha) [reply client/src/screens/BallotScreen.tsx:218]: If you were to combine the ternaries you get type refinement as you go:

```ts
contest.type === 'candidate'
  ? // must be candidate contest
  : // could be yesno or ms-either-neither
  contest.type == 'yesno'
  ? // must be yesno
  : // must be ms-either-neither
```

Comment (beausmith) [reply client/src/screens/BallotScreen.tsx:218]: I'm gonna make myself a new t-shirt:

I ❤️ nested terinaries

**Pull request merged**